gen/templates/resource.go.tmpl

```diff
@@ -1047,8 +1067,9 @@ func (r *{{.ResourceClassName}}Resource) Create(ctx context.Context, req resourc
 		set{{.ResourceClassName}}Id(ctx, data)
 	}
 
-	tflog.Debug(ctx, fmt.Sprintf("Create of resource aci_{{.ResourceName}} with id '%s'", data.Id.ValueString()))
+	set{{ .ResourceClassName }}Id(ctx, data)
```

Comment:
Should we still keep the create debug logging?
Is it possible just to move the log after the set?

```suggestion
	set{{ .ResourceClassName }}Id(ctx, data)
	tflog.Debug(ctx, fmt.Sprintf("Create of resource aci_{{.ResourceName}} with id '%s'", data.Id.ValueString()))
```

Reply:
Yes, done!